code/modules/mining/mine_items.dm

```diff
@@ -327,8 +328,29 @@
 	icon = 'icons/obj/lavaland/survival_pod.dmi'
 	icon_state = "fans"
 	name = "environmental regulation system"
+	desc = "A large machine releasing a constant gust of air.
 	anchored = 1
 	density = 1
+	var/arbitraryatmosblockingvar = TRUE
+
+/obj/structure/fans/tiny
+	name = "tiny fan"
+	desc = "A tiny fan, releasing a thin gust of air."
+	layer = TURF_LAYER + 0.8
+	density = 0
+	icon_state = "fan_tiny"
+
+/obj/structure/fans/New(loc)
+	..()
+	air_update_turf(1)
+
+/obj/structure/fans/Destroy()
+	arbitraryatmosblockingvar = FALSE
+	air_update_turf(1)
+	return ..()
+
+/obj/structure/fans/CanAtmosPass(turf/T)
+	return !arbitraryatmosblockingvar
```

Comment:
I'm pretty sure I need it for destroy so that when it's deleted the turf it's on becomes passable again. Like windows or blobs.